### PR TITLE
2021-12-17-bert_base_finnish_cased_fi

### PR DESCRIPTION
**Pull request by jsl-models**

### CODE REVIEW

File: docs/_posts/Ahmetemintek/2021-12-17-bert_base_finnish_cased_fi.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+nlp_pipeline = Pipeline(stages=[document_assembler, sentence_detector, tokenizer, embeddings])
+```
+```scala
+...
+val embeddings = BertEmbeddings.pretrained("bert_base_finnish_cased", "fi")
+      .setInputCols("sentence", "token")
+      .setOutputCol("embeddings")
+
+val pipeline = new Pipeline().setStages(Array(document_assembler, sentence_detector, tokenizer, embeddings))
+val data = Seq("Rakastan NLP: tä").toDF("text")
```
Comment (josejuanmartinez) [inline docs/_posts/Ahmetemintek/2021-12-17-bert_base_finnish_cased_fi.md:56]: Can we find a bigger example from Finnish wikipedia?

File: docs/_posts/Ahmetemintek/2021-12-17-bert_base_finnish_cased_fi.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+<div class="tabs-box" markdown="1">
+{% include programmingLanguageSelectScalaPythonNLU.html %}
+```python
+...
+embeddings = BertEmbeddings.pretrained("bert_base_finnish_cased", "fi") \
+      .setInputCols("sentence", "token") \
+      .setOutputCol("embeddings")
+
+nlp_pipeline = Pipeline(stages=[document_assembler, sentence_detector, tokenizer, embeddings])
```
Comment (josejuanmartinez) [inline docs/_posts/Ahmetemintek/2021-12-17-bert_base_finnish_cased_fi.md:47]: Part of the Python code is missing. Where is the fit / transform?

File: docs/_posts/Ahmetemintek/2021-12-17-bert_base_finnish_cased_fi.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+val data = Seq("Rakastan NLP: tä").toDF("text")
+val result = pipeline.fit(data).transform(data)
+```
+</div>
+
+## Results
+
+```bash
++--------------------+--------+
+|          embeddings|     token|
```
Comment (josejuanmartinez) [inline docs/_posts/Ahmetemintek/2021-12-17-bert_base_finnish_cased_fi.md:65]: Can we fix indentation here?

**Review by josejuanmartinez — CHANGES_REQUESTED** [review]
Hi @Ahmetemintek  Can you take a look at the requested changes? Thank you!

File: docs/_posts/Ahmetemintek/2021-12-17-bert_base_finnish_cased_fi.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+[Download](https://s3.amazonaws.com/community.johnsnowlabs.com/Ahmetemintek/bert_base_finnish_cased_fi_3.3.4_3.0_1639741264494.zip){:.button.button-orange.button-orange-trans.arr.button-icon}
+
+## How to use
+
+
+
+<div class="tabs-box" markdown="1">
+{% include programmingLanguageSelectScalaPythonNLU.html %}
+```python
+...
```
Comment (josejuanmartinez) [inline docs/_posts/Ahmetemintek/2021-12-17-bert_base_finnish_cased_fi.md:42]: Can we add the rest of the components? Document Assembler, SentenceDetector, Tokenizer...

File: docs/_posts/Ahmetemintek/2021-12-17-bert_base_finnish_cased_fi.md [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+embeddings = BertEmbeddings.pretrained("bert_base_finnish_cased", "fi") \
+      .setInputCols("sentence", "token") \
+      .setOutputCol("embeddings")
+
+nlp_pipeline = Pipeline(stages=[document_assembler, sentence_detector, tokenizer, embeddings])
+```
+```scala
+...
```
Comment (josejuanmartinez) [inline docs/_posts/Ahmetemintek/2021-12-17-bert_base_finnish_cased_fi.md:50]: Can we add the rest of the components? Document Assembler, SentenceDetector, Tokenizer...

File: docs/_posts/Ahmetemintek/2021-12-17-bert_base_finnish_cased_fi.md (hunk 1 above)
Comment (Ahmetemintek) [reply docs/_posts/Ahmetemintek/2021-12-17-bert_base_finnish_cased_fi.md]: I've updated input example.

File: docs/_posts/Ahmetemintek/2021-12-17-bert_base_finnish_cased_fi.md (hunk 2 above)
Comment (Ahmetemintek) [reply docs/_posts/Ahmetemintek/2021-12-17-bert_base_finnish_cased_fi.md:60]: I've updated python and scala codes.

File: docs/_posts/Ahmetemintek/2021-12-17-bert_base_finnish_cased_fi.md (hunk 4 above)
Comment (Ahmetemintek) [reply docs/_posts/Ahmetemintek/2021-12-17-bert_base_finnish_cased_fi.md]: I've updated python and scala codes.

File: docs/_posts/Ahmetemintek/2021-12-17-bert_base_finnish_cased_fi.md (hunk 5 above)
Comment (Ahmetemintek) [reply docs/_posts/Ahmetemintek/2021-12-17-bert_base_finnish_cased_fi.md:65]: I've updated python and scala codes.

**Review by josejuanmartinez — APPROVED** [review]
Looks good, thank you!

**Pull request merged**